spec/orm/activerecord_spec.rb

```diff
@@ -780,10 +780,12 @@ def filename
       Event.transaction do
         @event.image = stub_file('new.jpeg')
         @event.save
+        expect(File.exist?(public_path('uploads/new.jpeg'))).to be_truthy
```

Comment:
These expectations should be added as a separate test case, since they do not match with the subject of this case `it 'should not remove old file if transaction is rollback'`.